apps/dashboard/app/apps/ood_app_group.rb

```diff
@@ -68,6 +68,6 @@ def self.select(titles:[], groups:[])
   # then display other categories in alphabetical order
   def self.order(titles:[], groups:[])
       h = Hash[groups.map {|g| [g.title, g]}]
-      titles.concat(h.keys.sort).uniq.map { |t| h.has_key?(t) ? h[t] : nil }.compact 
+      titles.clone.concat(h.keys.sort).uniq.map { |t| h.has_key?(t) ? h[t] : nil }.compact
```

Comment:
Does this read better? I can't tell, but I feel like I use `concat` when I know I'm modifying the existing array, so saying `clone.concat` seems off to me.

```suggestion
      (titles + h.keys.sort).uniq.map { |t| h.has_key?(t) ? h[t] : nil }.compact
```

Reply:
LGTM. Updating.